humble/distribution.yaml

```diff
@@ -6128,11 +6128,15 @@ repositories:
       version: master
     status: maintained
   rplidar_ros:
+    doc:
+      type: git
+      url: https://github.com/Slamtec/rplidar_ros.git
```

Comment:
I notice that this URL is different than the `source` URL. These should ideally be the same, is there a strong reason to keep them different?

Reply:
The `source `URL should be changed to` https://github.com/Slamtec/rplidar_ros.git`. I close this PR and re-apply for PR after modification, is that okay?

Reply:
It looks like you made the change, thank you! Will you also be changing Iron and Rolling?

Reply:
Yes, I will change them in the future